webui/react/src/pages/ExperimentDetails/ExperimentCheckpoints.tsx

```diff
@@ -283,9 +283,9 @@ const ExperimentCheckpoints: React.FC<Props> = ({ experiment, pageRef }: Props)
         { signal: canceler.signal },
       );
       setTotal(response.pagination.total ?? 0);
-      if (!isEqual(response.checkpoints, checkpoints)) {
-        setCheckpoints(response.checkpoints);
-      }
+      setCheckpoints((cps) => {
+        return isEqual(response.checkpoints, cps) ? response.checkpoints : cps;
```

Comment:
just realized i got the ternary order wrong:
```suggestion
        return isEqual(response.checkpoints, cps) ? cps : response.checkpoints;
```

Reply:
Ah -- thanks!